src/dom-functions.js

```diff
@@ -8,29 +9,48 @@ export const initializePageHTML = () => {
     <section class="leaderboard">
       <div class="header">
         <h2>Recent scores</h2>
-        <button type="button">Refresh</button>
+        <button id="refresh-btn" type="button">Refresh</button>
       </div>
-      <ul class="scores">
-        <li>Name: 100</li>
-        <li>Name: 20</li>
-        <li>Name: 50</li>
-        <li>Name: 78</li>
-        <li>Name: 125</li>
-        <li>Name: 77</li>
-        <li>Name: 42</li>
+      <ul id="scoreboard">
       </ul>
     </section>
     <section class="add-score">
       <div class="header">
         <h2>Add your score</h2>
       </div>
       <form action="#">
-        <input type="text" name="name" id="name" placeholder="Your name">
-        <input type="number" name="score" id="score" placeholder="Your score">
+        <input type="text" name="name" id="name" placeholder="Your name" required>
+        <input type="number" name="score" id="score" placeholder="Your score" required>
         <input type="submit" value="Submit">
       </form>
     </section>
   </main>
   `;
   document.body.innerHTML = initialHTML;
+};
+
+export const initializeScoreboard = () => {
+  const scoreboard = document.getElementById('scoreboard');
```

Comment:
- [OPTIONAL] Prefer the use of camelCase for variable names.

Reply:
Hi, Thank you although scoreboard is one word